CMakeLists.txt

```diff
@@ -46,15 +46,24 @@ option(ENABLE_MKLGPU_BACKEND "" ON)
 if(ENABLE_MKLCPU_BACKEND)
   option(ENABLE_MKLCPU_THREAD_TBB "" ON)
 endif()
-option(ENABLE_CUBLAS_BACKEND "" OFF)
-
-option(ENABLE_CUSOLVER_BACKEND "" OFF)
 
+# blas
+option(ENABLE_CUBLAS_BACKEND "" OFF)
 option(ENABLE_ROCBLAS_BACKEND "" OFF)
+option(ENABLE_NETLIB_BACKEND "" OFF)
+
+# rand
 option(ENABLE_CURAND_BACKEND "" OFF)
 option(ENABLE_ROCRAND_BACKEND "" OFF)
+
+# lapack
+option(ENABLE_CUSOLVER_BACKEND "" OFF)
 option(ENABLE_ROCSOLVER_BACKEND "" OFF)
-option(ENABLE_NETLIB_BACKEND "" OFF)
+
+# dft
+option(ENABlE_CUFFT_BACKEND "enable the cuFFT backend for DFT" OFF)
```

Comment:
There's a sneaky lower case `l` here and below!

Reply:
addressed in https://github.com/oneapi-src/oneMKL/pull/284/commits/f03050fb1e0c24b3d947949b379984c1439fabcb and https://github.com/oneapi-src/oneMKL/pull/284/commits/e6bd53e78bbc209029e252e470b7d40712115515